ui/src/app/components/resource-card.tsx

```diff
@@ -150,7 +150,7 @@ class ResourceCard extends React.Component<Props, {}> {
           </FlexColumn>
           <div style={styles.cardFooter}>
             <div style={styles.lastModified} data-test-id='last-modified'>
-              Last Modified: {formatWorkspaceResourceDisplayDate(resource.lastModifiedEpochMillis)}</div>
+              Last Modified: {displayDateWithoutHours(resource.lastModifiedEpochMillis)}</div>
```

Comment:
Identical except for a comma: `Last Modified: Nov 15 2021` becomes `Last Modified: Nov 15, 2021`